_episodes/01-motivation.md

```diff
@@ -7,118 +7,108 @@ questions:
+mylib-1.2.4_6.10.07.tgz         somecode_CP_24.5.07.tgz
+mylib-1.2.5_23.4.08.tgz         somecode_CP_25.5.07.tgz
+mylib-1.2.5_25.5.07.tgz         somecode_CP_29.5.07.tgz
+mylib-1.2.5_6.6.07.tgz          somecode_CP_30.5.07.tgz
+mylib-1.2.5_bexc.tgz            somecode_CP_6.10.07.tgz
+mylib-1.2.5_d0.tgz              somecode_CP_6.6.07.tgz
+mylib-1.3.0_4.4.08.tgz          somecode_CP_8.6.07.tgz
+mylib-1.3.1_4.4.08.tgz          somecode_KT.tgz
+mylib-1.3.2_22.4.08.tgz         somecode_PI1_2007.tgz
+mylib-1.3.2_4.4.08.tgz          somecode_PI_2007.tgz
+mylib-1.3.2_5.4.08.tgz          somecode_PI2_2007.tgz
+mylib-1.3.3_1.5.08.tgz          somecode_PI_CP_18.3.07.tgz
+mylib-1.3.3_20.5.08.tgz         somecode_11.5.08.tgz
+mylib-1.3.3_tstrm_27.6.08.tgz   somecode_15.4.08.tgz
+mylib-1.3.3_wk_10.8.08.tgz      somecode_17.6.09_unfinished.tgz
+mylib-1.3.3_wk_11.8.08.tgz      somecode_19.7.09.tgz
+mylib-1.3.3_wk_13.8.08.tgz      somecode-20.7.09.tgz
+...
+```
 
 ---
 
-## Why Git?
+## Compare with Dropbox or Google Drive
+
+- Document/code is in one place, no need to email snapshots.
+- Possible to get old versions but in a much less useful way - snapshots of files, not directories.
+
+---
 
+## We will use [Git](https://git-scm.com) to record snapshots of our work - why Git?
 
-- Distributed
-- Other people use it
-- Merging and branching is easy and simple.
+- Easy to set up.
+- Very popular: chances are high you will need to contribute to somebody else's code which is tracked with Git.
+- Distributed: good backup, no single point of failure, you can track and clean-up changes offline, simplifies collaboration model for open-source projects.
 - Important platforms such as [GitHub](https://github.com), [GitLab](https://gitlab.com), and [Bitbucket](https://bitbucket.org)
   build on top of Git.
 - Many platforms build on top of [GitHub](https://github.com).
 - Sharing software and data is getting popular and required in research context
   and [GitHub](https://github.com) is a popular platform for sharing software.
-- Can start using now, no need to configure a server 
-- *"Git is a four-handle, dual boiler espresso machine, not instant coffee."* [citation needed]
+- However, *"Git is a four-handle, dual boiler espresso machine, not instant coffee."* [citation needed]
 
 ---
 
 ## Why Git and not [Subversion](https://subversion.apache.org)?
 
-- Subversion is centralized (one server, many clients).
-- Git is distributed and supports any workflow.
-- Distributed version control enables to:
-    - implement code review
-    - decentralize access control
-    - decentralize bug exposure
-    - commit changes without network
-    - edit changes after they have been committed
+- Subversion is centralized (one server, many clients) and requires setting up and maintaining a server.
+- You cannot easily clean-up your recorded snapshots (commits) before you share them.
+- Not easy to get contributions from external contributors.
 
 ---
 
```

Comment:
Mercurial: many git concepts still apply.  For that matter, most important lesson is how and why to use version control, which applies to any system with some changes.